cloudformation/anghammarad.template.yaml

```diff
@@ -66,6 +79,9 @@ Resources:
         Stack: !FindInMap [ Constants, Stack, Value ]
         App: !FindInMap [ Constants, App, Value ]
         Stage: !Ref Stage
+      DeadLetterQueue:
```

Comment:
The serverless application model is quite clever here; it automatically adds an IAM policy which allows the Lambda to send unprocessable messages to the queue